exporter/awsemfexporter/README.md

```diff
@@ -25,6 +25,7 @@ The following exporter configuration parameters are supported.
 | `max_retries`     | Maximum number of retries before abandoning an attempt to post data.   |    1    |
 | `dimension_rollup_option`| DimensionRollupOption is the option for metrics dimension rollup. Three options are available. |"ZeroAndSingleDimensionRollup" (Enable both zero dimension rollup and single dimension rollup)| 
 | `resource_to_telemetry_conversion` | "resource_to_telemetry_conversion" is the option for converting resource attributes to telemetry attributes. It has only one config onption- `enabled`. For metrics, if `enabled=true`, all the resource attributes will be converted to metric labels by default. See `Resource Attributes to Metric Labels` section below for examples. | `enabled=false` | 
+| `output_destination` | "output_destination" is an option to specify the EMFExporter output. Currently, two options are available. "CloudWatch" or "stdout" | `CloudWatch` | 
```

Comment:
```suggestion
| `output_destination` | "output_destination" is an option to specify the EMFExporter output. Currently, two options are available. "cloudwatch" or "stdout" | `cloudwatch` | 
```

All lowercase seems more consistent with the config keys

Reply:
Thanks. I make the change accordingly.